thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/task/TaskRunnerFactory.java

```diff
@@ -59,16 +56,6 @@ public static TaskRunner getTaskRunnerFromTaskType(TaskType taskType) {
       case MONITOR:
         taskRunner = new MonitorTaskRunner();
         break;
-      case DATA_COMPLETENESS:
```

Comment:
ANOMALY_DETECTION and MERGE tasks are not used.

Reply:
Removed.